### PR TITLE
Amazon Cloud Player script

### PR DESCRIPTION
**Pull request by SteVwonder**
Sorry this took so long.  I integrated all of the suggested changes from #1, except the national variants.

Once this PR is merged, either you or I should open an issue about the variants.  I can probably look into it this weekend.

### CODE REVIEW

**Comment by jiri-janousek** [comment]
Thanks, @SteVwonder. I've added this code review to the roadmap for July 2015 tiliado/nuvolaplayer.project#3

File: integrate.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        track.title = songDetails.getElementsByClassName("title")[0].innerText;
+        track.artist = songDetails.getElementsByClassName("artistLink")[0].innerText;
+
+        var albumImage = playerRoot.getElementsByClassName("albumImage")[0];
+        track.artLocation = albumImage.src;
+        if (track.album === null)
+            track.album = albumImage.title;
+    } catch (e) {
+        console.log("Failed to get track info");
+        console.log(e.message);
```
Comment (jiri-janousek) [inline integrate.js]: These logging commands are pretty annoying. Could you comment them out with Geany-style toggle comments "//~"?

**Comment by jiri-janousek** [comment]
Hello. I'm ready to merge you branch as soon as you fix one minor issue (see inline comment). However, I won't create packages until the national variants are supported. Will you take a look at it or should I take care of it?

**Comment by SteVwonder** [comment]
I commented out the console logs.  I don't have time the next two weeks to look into the variants, but I can the first week of August.

**Comment by jiri-janousek** [comment]
Great :+1: I will merge this PR later today. You can then create a new PR for national variants when it's ready.

**Pull request merged**